### PR TITLE
[ADD] mrp_brewing / warehousekeeper number configuration

### PR DESCRIPTION
**Pull request by vvrossem**
New feature : warehousekeeper number configuration in `Configuration > Paramètres généraux > Configurer les données de la société > Numéro d'entrepositaire`
This number is used on the following reports : 
- raw materials
- finished products
- brew register

Room for improvement : format the warehousekeeper number according to the belgian law

### CODE REVIEW

File: mrp_brewing/models/company.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+
+from openerp import api, fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    warehousekeeper_nb = fields.Char(
```
Comment (houssine78) [inline mrp_brewing/models/company.py]: just wondering if nb is an appropriate shortname. @robinkeunen ok for you ?

**Review by houssine78 — APPROVED** [review]

File: mrp_brewing/models/company.py (hunk 1 above)
Comment (vvrossem) [reply mrp_brewing/models/company.py]: you are right, cambridge dictionary recommends _no_ usage

Comment (vvrossem) [reply mrp_brewing/models/company.py]: done

**Comment by houssine78** [comment]
@vvrossem ok for me. can be merged as soon it passes the test. make sure you put the PR link in the related task. Thanks !

**Comment by robinkeunen** [comment]
> ok for me. can be merged as soon it passes the test. 

The tests are not scheduled on this repo and branch so you can go forward with it.

**Review by robinkeunen — APPROVED** [review]
You could blacken this moduke _en passant_.

**Comment by houssine78** [comment]
> 
> 
> > ok for me. can be merged as soon it passes the test.
> 
> The tests are not scheduled on this repo and branch so you can go forward with it.

I meaned manual test from our BA :)

**Comment by robinkeunen** [comment]
Ah yes, _those tests_ 🤭

**Comment by vvrossem** [comment]
It passed BA tests :) 
I've just _blacken_ the module

**Pull request merged**